include/net/ieee802154_radio.h

```diff
@@ -54,6 +54,41 @@ struct ieee802154_filter {
 /* @endcond */
 };
 
+/** IEEE802.15.4 driver configuration types. */
+enum ieee802154_config_type {
+	/** Indicates how radio driver should set Frame Pending bit in ACK
+	 *  responses for Data Requests. If enabled, radio driver should
+	 *  determine whether to set the bit or not based on the information
+	 *  provided with ``set_ack_data`` function. Otherwise, Frame Pending
+	 *  bit should be set to ``1`` (see IEEE Std 802.15.4-2006, 7.2.2.3.1).
+	 */
+	IEEE802154_CONFIG_AUTO_ACK_FRAME_PENDING_BIT,
```

Comment:
I wonder if coalescing 'FRAME_PENDING_BIT' to 'FPB' would be relevant, as long as it is properly documented. At least that would reduce the names here and below in config struct as well.

Reply:
Hmm I agree, it was quite inconvenient to use such long names when I used this API with OpenThread. The docs are pretty clear here IMO, so I guess we could use an acronym in the config name. @jukkar What'd you say?

Reply:
Ok for me.

Reply:
Done